x/meshsecurity/types/params.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+// todo: should be generated from protobuf
+// todo: also add new messages to update params
+type Params struct{}
```

Comment:
I would just store the authority. And maybe a "max max cap", or "max providers" (both optional)
So we couldn't have 100 providers, or no provider with 200 million cap.

I mean, anything would have to go through gov vote, but maybe such "hard to change, only on code migration updatable" limits may be a nice second layer of defense against some gov typo (miscounted 0s or such)

Reply:
👍 good idea!